internal/provider/trigger_resource_test.go

```diff
@@ -69,6 +69,7 @@ func TestAcc_TriggerResourceUpgradeFromVersion014(t *testing.T) {
 			{
 				ProtoV5ProviderFactories: testAccProtoV5MuxServerFactory,
 				Config:                   config,
+				PlanOnly:                 true,
```

Comment:
This was always the recommendation of how this test should be shaped but the way things were working broke it 🙃 

Restored to the documented approach